contrib/firebaseio/src/main/java/com/google/cloud/dataflow/contrib/firebase/io/FirebaseSource.java

```diff
@@ -84,7 +93,7 @@
 
     FirebaseReader(
         FirebaseSource<T> source,
-        FirebaseCheckpoint<T> checkpoint) throws NoSuchAlgorithmException{
+        FirebaseCheckpoint<T> checkpoint) throws NoSuchAlgorithmException {
       this.source = source;
       this.digest = MessageDigest.getInstance("MD5");
```

Comment:
per comments elsewhere -- would probably catch here and wrap it as an IOException or a RuntimeException.